Project.toml

```diff
@@ -1,6 +1,6 @@
 name = "Oceananigans"
 uuid = "9e8cae18-63c1-5223-a75c-80ca9d6e9a09"
-version = "0.66.3"
+version = "0.67.1"
```

Comment:
why is this considered change? version is 0.67.1 already?
@simone-silvestri perhaps bring this branch up to date with main?

Reply:
Done!

Reply:
Changes are done! I'll merge